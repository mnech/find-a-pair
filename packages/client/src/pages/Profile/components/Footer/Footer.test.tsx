import React from 'react';
import { render } from '@testing-library/react';
import { Footer } from './index';

describe('Footer profile component', () => {
  describe('should be rendered correctly Footer component', () => {
    test('default', () => {
      const setIsEditData = jest.fn();
      const setIsEditPassword = jest.fn();

      const { asFragment } = render(
        <Footer
          setIsEditData={setIsEditData}
          setIsEditPassword={setIsEditPassword}
        />,
      );
      expect(asFragment()).toMatchSnapshot();
    });
  });
});
