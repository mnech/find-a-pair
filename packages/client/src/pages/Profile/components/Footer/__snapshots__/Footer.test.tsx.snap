// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer profile component should be rendered correctly Footer component default 1`] = `
<DocumentFragment>
  <footer
    class="footer"
  >
    <button
      class="btn btn-link"
      type="button"
    >
      Изменить данные
    </button>
    <button
      class="btn btn-link"
      type="button"
    >
      Изменить пароль
    </button>
    <button
      class="danger btn btn-link"
      type="button"
    >
      Выйти
    </button>
  </footer>
</DocumentFragment>
`;
